server/fakeData/fakeDataHelpers.js

```diff
@@ -40,11 +40,11 @@ const createFakeCategories = (n) => {
   return categories;
 }
 
-const fillCategories = (n) => {
+const fillCategories = async (n) => {
```

Comment:
I like your use of async/await here. It is a good choice to help prevent an overload of database queries. Great job!

Reply:
Thank you for the feedback!